framework/src/kernels/VectorKernel.C

```diff
@@ -99,24 +99,25 @@ VectorKernel::computeJacobian()
 }
 
 void
-VectorKernel::computeOffDiagJacobian(MooseVariableFEBase & jvar)
+VectorKernel::computeOffDiagJacobian(const unsigned int jvar_num)
```

Comment:
const unsigned int?

Reply:
This is to prevent some deviant from coming into the method body and reassigning `jvar_num`.